tests/resources/test_issue_type_scheme_associations.py

```diff
@@ -0,0 +1,21 @@
+from tests.conftest import JiraTestCase
+
+
+class IssueTypeSchemeAssociationTests(JiraTestCase):
+    def test_scheme_associations(self):
+        allSchemes = self.jira.issue_type_schemes()
```

Comment:
```suggestion
        all_schemes = self.jira.issue_type_schemes()
```

is the preferred way for variables, older code isn't as consistent, but let's try and be consistent with the newly added stuff